README.md

```diff
@@ -10,4 +10,239 @@ TODO: description
 
 ## API Routes
 
-TODO: document API routes
+### Shared Account
+
+TODO: Jeremiah
+
+### Insurance Card
+
+#### `GET /api/insuranceCards?owner=USERNAME` - Get insurance cards by owner
+
+**Returns**
+
+- An array of insurance cards created by user with username `owner` in alphabetical order with respect to their purpose
+
+**Throws**
+
+- `400` if `owner` is not given
+- `404` if `owner` is not a recognized username of any user
```

Comment:
Should we also check if owner is currently logged in user? Logic being that users should only be able to access their own info